test_runner/blog/tests.py

```diff
@@ -1077,55 +1079,55 @@ def testPasswordRepeat(self):
-            response = self.client.post(reverse('users.user_login'), post_data, follow=True)
+        # log in
+        self.client.logout()
+        post_data = dict(username='plain', password='Password1')
+        response = self.client.post(reverse('users.user_login'), post_data, follow=True)
 
-            # assert we are being taken to our new password page
-            self.assertTrue('form' in response.context)
-            self.assertTrue('new_password' in response.context['form'].fields)
+        # assert we are being taken to our new password page
+        self.assertTrue('form' in response.context)
+        self.assertTrue('new_password' in response.context['form'].fields)
 
-            # try to go to a different page
-            response = self.client.get(reverse('blog.post_list'), follow=True)
+        # try to go to a different page
+        response = self.client.get(reverse('blog.post_list'), follow=True)
 
-            # redirected again
-            self.assertTrue('form' in response.context)
-            self.assertTrue('new_password' in response.context['form'].fields)
+        # redirected again
+        self.assertTrue('form' in response.context)
+        self.assertTrue('new_password' in response.context['form'].fields)
 
-            # ok, set our new password
-            post_data = dict(old_password='Password1', new_password='Password1', confirm_new_password='Password1')
-            response = self.client.post(reverse('users.user_newpassword', args=[0]), post_data)
+        # ok, set our new password
+        post_data = dict(old_password='Password1', new_password='Password1', confirm_new_password='Password1')
+        response = self.client.post(reverse('users.user_newpassword', args=[0]), post_data)
 
-            # we should get a failure that our new password is a repeat
-            self.assertTrue('confirm_new_password' in response.context['form'].errors)
+        # we should get a failure that our new password is a repeat
+        self.assertTrue('confirm_new_password' in response.context['form'].errors)
 
-            # use a different password
-            post_data = dict(old_password='Password1', new_password='Password2', confirm_new_password='Password2')
-            response = self.client.post(reverse('users.user_newpassword', args=[0]), post_data)
+        # use a different password
+        post_data = dict(old_password='Password1', new_password='Password2', confirm_new_password='Password2')
+        response = self.client.post(reverse('users.user_newpassword', args=[0]), post_data)
 
-            # should be redirected to the normal redirect page
-            self.assertEquals(302, response.status_code)
-            self.assertTrue(response['location'].find(reverse('blog.post_list')) > 0)
+        # should be redirected to the normal redirect page
+        self.assertEquals(302, response.status_code)
+        self.assertIn(reverse('blog.post_list'), response['location'])
```

Comment:
Prior to Django 1.9, redirect URLs were always made absolute - so in a test like this, `response['location']` may be an absolute URL ending with `reverse('blog.post_list')`, or it they may be equal (Django 1.9)